packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -538,55 +540,41 @@ void main() {
     editable.selection = const TextSelection(baseOffset: 29, extentOffset: 29);
 
     final EditableTextState editableTextState = tester.firstState(find.byType(EditableText));
-    editableTextState.updateFloatingCursor(
-      RawFloatingCursorPoint(state: FloatingCursorDragState.Start),
-    );
-    editableTextState.updateFloatingCursor(
-      RawFloatingCursorPoint(
-        state: FloatingCursorDragState.Update,
-        offset: const Offset(20, 20),
-      ),
-    );
+    editableTextState.updateFloatingCursor(RawFloatingCursorPoint(state: FloatingCursorDragState.Start));
+    editableTextState.updateFloatingCursor(RawFloatingCursorPoint(state: FloatingCursorDragState.Update, offset: const Offset(20, 20)));
     await tester.pump();
 
     expect(editable, paints
-      ..rrect(
-        rrect: RRect.fromRectAndRadius(
-          Rect.fromLTRB(464.6666564941406, 2.0833332538604736, 466.6666564941406, 14.083333015441895),
-          const Radius.circular(2.0),
-        ),
+      ..rrect(rrect: RRect.fromRectAndRadius(
```

Comment:
Quite a lot of needless code reformatting here. In this case (and others) the new version matches what's explicitly called out as "bad" in the style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation